lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
`require "action_controller"` do not require `ActionController::Base` so it do not trigger the `on_load` hook. All Rails frameworks require you to load the root file to setup all the hooks and common dependencies. What @josevalim is said is If requiring `action_controller` is loading `ActionController::Base` it is a bug on Rails and needs to be fixed.